Mathlib/NumberTheory/LSeries/SumCoeff.lean

```diff
@@ -153,3 +160,222 @@ theorem LSeries_eq_mul_integral_of_nonneg (f : ℕ → ℝ) {r : ℝ} (hr : 0 
   LSeries_eq_mul_integral' _ hr hs <| hO.congr_left fun _ ↦ by simp [_root_.abs_of_nonneg (hf _)]
 
 end integralrepresentation
+
+noncomputable section residue
+
+variable {f : ℕ → ℂ} {l : ℂ} (hlim : Tendsto (fun n : ℕ ↦ (∑ k ∈ Icc 1 n, f k) / n) atTop (𝓝 l))
```

Comment:
Using `variable` with Prop-valued hypotheses tends to be a bit clumsy; you have to `include` them every time, so it's probably simpler just to reproduce the hypothesis in each lemma